src/messages/variant.rs

```diff
@@ -147,10 +148,10 @@ impl Debug for Variant {
                 .finish(),
             Self::UserMessage(payload) => write!(f, "UserMessage({:10})", HexFmt(payload)),
             Self::NodeApproval(payload) => write!(f, "NodeApproval({:?})", payload),
-            Self::Sync(shared_state) => f
+            Self::Sync { section, .. } => f
                 .debug_struct("Sync")
-                .field("elders_info", shared_state.sections.our())
-                .field("section_key", shared_state.our_history.last_key())
+                .field("elders_info", section.elders_info())
+                .field("section_key", section.chain().last_key())
```

Comment:
shall the network (or some info of it) be logged as well?

Reply:
I'd say that probably no. We weren't logging it before this PR and also we are probably soon going to change the way we keep `network` up to date. It's not decided yet, but It's possible it won't be part of `Sync` anymore.